TangemStaking/Services/StakingAPIProvider/StakingAPIProvider.swift

```diff
@@ -17,4 +17,5 @@ public protocol StakingAPIProvider {
 
     func patchTransaction(id: String) async throws -> TransactionInfo
     func submitTransaction(hash: String, signedTransaction: String) async throws
+    func submitHash(_ hash: String, for transactionId: String) async throws
```

Comment:
```suggestion
    func submitTransaction(hash: String, transactionId: String) async throws
```
я бы так продолжил, а то вот такие `for` как сложно с места вызова понимать